subproc.cc

```diff
@@ -535,9 +535,13 @@ pid_t cloneProc(uintptr_t flags, int exit_signal) {
 	    .stack = 0,
 	    .stack_size = 0,
 	    .tls = 0,
+#ifdef CLONE_ARGS_SIZE_VER1
```

Comment:
let's just leave out all the 0 fields (will be zero-initialized anyhow).

Reply:
I thought this would work too, but gcc has a warning (which we treat as an error) for missing fields. So we can do this nasty thing if you prefer? :)

```
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wmissing-field-initializers"
...
#pragma GCC diagnostic pop
```

Reply:
Right, we use the GCC extension here since we're not on C++20 yet :)
let's do:
```
clone_args = {};
clone_args.flags = (uint64_t)flags;
clone_args.exit_signal = (uint64_t)exit_signal;
```